_layouts/home.html

```diff
@@ -0,0 +1,227 @@
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <div class="table-holder mt-100 mb-80" id="specs">
+                    <div class="swiper-container">
+                        <div class="table-details swiper-wrapper">
+                            <div class="single-column swiper-slide">
+                                <ul>
+                                    <li>LAYER / PROTOCOL</li>
+                                    <li>Application Layer</li>
+                                    <li>Data Sync</li>
+                                    <li>Secure Transport</li>
+                                    <li>Transport Privacy</li>
+                                    <li>P2P Overlay</li>
+                                    <li>Data Sync</li>
+                                    <li></li>
+                                    <li>Trust Establishment</li>
+                                </ul>
+                            </div>
+                            <div class="single-column swiper-slide">
+                                <ul>
+                                    <li>PURPOSE</li>
+                                    <li>End user semantics</li>
+                                    <li>Data consistency</li>
+                                    <li>Confidentiality, PFS, etc</li>
+                                    <li>Transport and metadata protection</li>
+                                    <li>Overlay Routing, NAT traversal</li>
+                                    <li>Data consistency</li>
+                                    <li></li>
+                                    <li>Establishing end-to-end trust</li>
+                                </ul>
+                            </div>
+                            <div class="single-column swiper-slide">
+                                <ul>
+                                    <li>EXAMPLES</li>
+                                    <li>1:1 chat, group chat</li>
+                                    <li>MVDS, BSP</li>
+                                    <li>Double Ratchet, MLS</li>
+                                    <li>Whisper, TOR, Mixnet</li>
+                                    <li>devP2P, libP2P</li>
+                                    <li>MVDS, BSP</li>
+                                    <li></li>
+                                    <li>TOFU, web of trust</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="swiper-pagination"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="team" id="who">
```

Comment:
Just thinking out loud: do we need to have this section? It might be better to focus on the project and less on individuals. Also because there are more people who are tangentially involved.

@decanus @j-zerah what do you think?

Reply:
entirely up to you guys. I think it is nice to put some faces to the projects and humanize it a bit. It is also a nice way for people to get in touch directly if they want to.

Reply:
I'm OK keeping it as is, I suppose it's straightforward to add people later when we want to